helm-charts/common/lvm-serving/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: lvm-serving
```

Comment:
There are many lvm variant in the GenAIComps/lvm, it would be better to rename this as something like video-llama-lvm to make it specific.

Reply:
Done.